src/fosslight_util/write_excel.py

```diff
@@ -34,9 +34,10 @@
 def write_excel_and_csv(filename_without_extension, sheet_list, ignore_os=False, extended_header={}):
     success = True
     error_msg = ""
-    success_csv = True
+    success_csv = False
```

Comment:
If the default value of success_csv is False, this error message, `"[Error] Writing csv:"`, will always be displayed in Windows.

Reply:
And in the case of windows, the first return value is always False.